Makefile

```diff
@@ -261,12 +261,11 @@ fuzz:
 #### CLI release related ####
 
 VERSION ?= dev
-# Default to a dummy version 0.0.1.rc1, which is always lower than a real release.
-# This must be in the form of [0-255].[0-255].[0-65535] plus optional fourth element which will be ignored.
-# We use the fourth field to represent the rc portion of release tag (e.g. rc1 of 1.0.0-rc2).
+# Default to a dummy version 0.0.1.1, which is always lower than a real release.
+# Legal version values should look like 'x.x.x.x' where x is an integer from 0 to 65534.
 # https://learn.microsoft.com/en-us/windows/win32/msi/productversion?redirectedfrom=MSDN
 # https://stackoverflow.com/questions/9312221/msi-version-numbers
-MSI_VERSION ?= 0.0.1.rc1
+MSI_VERSION ?= 0.0.1.1
```

Comment:
@codefromthecrypt notice that candle.exe does not support 0.0.1.x with x being text. It *must* be an integer or it will cause an error.